app/controllers/wiki_controller.rb

```diff
@@ -133,6 +134,7 @@ def edit
   end
 
   def new
+    @is_new = true
```

Comment:
This works -- but there's maybe a more compact way to do it; we could skip this line, and in the view, just do

```ruby
if params[:action] == "new" && params[:controller] == "wiki"
```

What do you think?